processing/src/main/java/org/apache/druid/segment/serde/ComplexMetrics.java

```diff
@@ -45,11 +44,4 @@ public static void registerSerde(String type, ComplexMetricSerde serde)
     }
     complexSerializers.put(type, serde);
   }
-
-  public static void registerSerde(String type, Supplier<ComplexMetricSerde> serdeSupplier)
```

Comment:
@jon-wei this method was added on purpose to be used in other modules. See #7058.